ci-automation/release.sh

```diff
@@ -64,14 +74,103 @@ function _inside_mantle() {
 
     source ci-automation/ci_automation_common.sh
     source sdk_container/.repo/manifests/version.txt
+    CHANNEL="$(get_git_channel)"
+    VERSION="${FLATCAR_VERSION}"
+    azure_profile_config_file=""
+    secret_to_file azure_profile_config_file "${AZURE_PROFILE}"
+    azure_auth_config_file=""
+    secret_to_file azure_auth_config_file "${AZURE_AUTH_CREDENTIALS}"
+    aws_credentials_config_file=""
+    secret_to_file aws_credentials_config_file "${AWS_CREDENTIALS}"
+    aws_marketplace_credentials_file=""
+    secret_to_file aws_marketplace_credentials_file "${AWS_MARKETPLACE_CREDENTIALS}"
+    gcp_json_key_path=""
+    secret_to_file gcp_json_key_path "${GCP_JSON_KEY}"
+    google_release_credentials_file=""
+    secret_to_file google_release_credentials_file "${GOOGLE_RELEASE_CREDENTIALS}"
+
+    for platform in aws azure; do
+      for arch in amd64 arm64; do
+        # Create a folder where plume stores flatcar_production_ami_*txt and flatcar_production_ami_*json
+        # for later push to bincache
+        rm -rf "${platform}-${arch}"
+        mkdir "${platform}-${arch}"
+        cd "${platform}-${arch}"
+
+        export product="${CHANNEL}-${arch}"
+        pid=$(jq -r ".[env.product]" < product-ids.json)
```

Comment:
```suggestion
        pid=$(jq -r ".[env.product]" product-ids.json)
```
This here also works, or?

Reply:
Yep, that works too - TIL thanks!